lib/visualization/process_primer_sets_with_lineages.sh

```diff
@@ -23,7 +23,7 @@ fi
 variants_bed="$$_variants.bed"
 variants_counts_bed="$$_variants_with_counts.bed";
 echo "$(date +'%b %d %H:%M:%S') - starting DB fetch"
-echo "\"$(dirname "$0")/extract_all_variants.sh\" \"$cutoff_date\" > \"$variants_bed\";"
+#echo "\"$(dirname "$0")/extract_all_variants.sh\" \"$cutoff_date\" > \"$variants_bed\";"
```

Comment:
remove unless there is a good reason to keep this...

Reply:
Agreed, this was when I was making sure the command is right, it's not needed anymore.